go.mod

```diff
@@ -475,17 +475,22 @@ require (
 )
 
 require (
-	github.com/anthropics/anthropic-sdk-go v1.11.0
+	github.com/anthropics/anthropic-sdk-go v1.4.0
 	github.com/brianvoe/gofakeit/v7 v7.7.1
 	github.com/coder/agentapi-sdk-go v0.0.0-20250505131810-560d1d88d225
+	github.com/coder/aibridge v0.0.0
 	github.com/coder/aisdk-go v0.0.9
 	github.com/coder/preview v1.0.4
+	github.com/dgraph-io/ristretto/v2 v2.3.0
 	github.com/fsnotify/fsnotify v1.9.0
 	github.com/go-git/go-git/v5 v5.16.2
 	github.com/icholy/replace v0.6.0
-	github.com/mark3labs/mcp-go v0.32.0
+	github.com/mark3labs/mcp-go v0.38.0
 )
 
+// aibridge-related deps and directives. // TODO: replace with published tag.
+replace github.com/coder/aibridge v0.0.0 => /home/coder/aibridge
```

Comment:
I'm hoping to have the remaining PRs in `coder/aibridge` wrapped up and a tag published tomorrow.